README.md

```diff
@@ -179,3 +179,17 @@ Note that the contents of those file will be overidden on each run.
 
 We generate a self-signed wildcard certificate to use for all the ingress
 controllers.
+
+### How to build docker image with the right tag
```

Comment:
I'd put all this in a section dedicated to kites

Reply:
I agree, I am doing the section now